curriculum/challenges/english/25-front-end-development/quiz-css-flexbox/66ed8fe7f45ce3ece4053eb2.md

```diff
@@ -514,3 +514,445 @@ Dynamically adjusting distribution and alignment of items for different viewport
+
+---
+
+The spacing between flex items.
+
+#### --answer--
+
+The ability of a flex item to grow relative to others.
+
+### --question--
+
+#### --text--
+
+What is the initial value of the `flex-wrap` property?
+
+#### --distractors--
+
+`wrap`
+
+---
+
+`wrap-reverse`
+
+---
+
+`inherit`
+
+#### --answer--
+
+`nowrap`
+
+### --question--
+
+#### --text--
+
+Which property is used to define the space between flex items?
+
+#### --distractors--
+
+`align-items`
+
+---
+
+`flex-wrap`
+
+---
+
+`flex-direction`
+
+#### --answer--
+
+`gap`
+
+### --question--
+
+#### --text--
+
+What value of `justify-content` aligns items to the start of the main axis?
+
+#### --distractors--
+
+`center`
+
+---
+
+`space-between`
+
+---
+
+`space-around`
+
+#### --answer--
+
+`flex-start`
+
+### --question--
+
+#### --text--
+
+What value of `align-content` spaces items evenly with equal space around them?
+
+#### --distractors--
+
+`space-between`
+
+---
+
+`flex-start`
+
+---
+
+`center`
+
+#### --answer--
+
+`space-around`
+
+### --question--
+
+#### --text--
+
+What does the `order` property do in a flex container?
+
+#### --distractors--
+
+Defines the alignment of items along the cross axis.
+
+---
+
+Controls the wrapping behavior of flex items.
+
+---
+
+Sets the spacing between flex items.
+
+#### --answer--
+
+Specifies the order of the flex items.
+
+### --question--
+
+#### --text--
+
+Which property allows flex items to shrink if necessary?
+
+#### --distractors--
+
+`flex-grow`
+
+---
+
+`flex-wrap`
+
+---
+
+`align-items`
+
+#### --answer--
+
+`flex-shrink`
+
+### --question--
+
+#### --text--
+
+What value of `align-content` aligns items to the start of the cross axis?
+
+#### --distractors--
+
+`center`
+
+---
+
+`space-between`
+
+---
+
+`space-around`
+
+#### --answer--
+
+`flex-start`
+
+### --question--
+
+#### --text--
+
+What value of `align-content` aligns items to the end of the cross axis?
+
+#### --distractors--
+
+`flex-start`
+
+---
+
+`center`
+
+---
+
+`space-between`
+
+#### --answer--
+
+`flex-end`
+
+### --question--
+
+#### --text--
+
+What value of `justify-content` spaces items evenly with equal space around them?
+
+#### --distractors--
+
+`space-between`
+
+---
+
+`flex-start`
+
+---
+
+`center`
+
+#### --answer--
+
+`space-around`
+
+### --question--
+
+#### --text--
+
+What value of `justify-content` spaces items evenly with equal space between them?
+
+#### --distractors--
+
+`space-around`
+
+---
+
+`flex-start`
+
+---
+
+`center`
+
+#### --answer--
+
+`space-between`
+
+### --question--
+
+#### --text--
+
+What value of `align-items` stretches items to fill the container?
+
+#### --distractors--
+
+`flex-start`
+
+---
+
+`flex-end`
+
+---
+
+`center`
+
+#### --answer--
+
+`stretch`
+
+### --question--
+
+#### --text--
+
+What value of `align-content` spaces items evenly with equal space between them?
+
+#### --distractors--
+
+`space-around`
+
+---
+
+`flex-start`
+
+---
+
+`center`
+
+#### --answer--
+
+`space-between`
```

Comment:
same issue here

was this covered in the lectures or workshop?

If this was covered please provide the link in the course where it was covered in your reply and we can keep this question. Otherwise, this needs to be replaced